spec/frodata/abstract_client_spec.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe FrOData::AbstractClient do
+  subject { described_class }
+
+  it { should < FrOData::Concerns::Base }
```

Comment:
I am having trouble figuring out what this syntax evaluates to, but I am guessing it validates that's it's included in the class?

Reply:
it s one of the weirdness of rspec :) it compares it to the subject

Reply:
and subject is the describe_class . (what is specified after described)